index.js

```diff
@@ -1,7 +1,59 @@
 // Iteration 1: Names and Input
+let hacker1 = "Ivan";
+let hacker2 = "Jose";
 
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
-
+if (hacker1.length > hacker2.length) {
+    console.log (`The driver has the longest name, it has ${hacker1.length} characters.`)
+  } else if (hacker2.length > hacker1.length){
+    console.log (`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+  } else {
+     console.log (`Wow, you both have equally long names, ${hacker1.length} characters!`)
+  }
 
 // Iteration 3: Loops
+// 3.1
+
+let result = "";
+hacker1 = `${hacker1}`;
+
+let j = 0;
+
+for (let i = 0; i <= 3; i++){
+  const x = hacker1[i];
+
+  if (j === 4) {
+      result = `${result} `
+      j = 0
+  }
+
+  j++
+
+  result = `${result} ${x}`
+}
+
+console.log(result.toUpperCase())
```

Comment:
Ok!, es una solucion pero solo para nombres con 4 letras.... y en el fondo esa no es la idea del ejercicio, lo suyo seria cualquier nombre yo te lo paso y tu me lo pones separado por espacios y en mayusculas

Para hacerlo 100% tenemos que saber cual es la longitud de la cadena en cuestion, eso si lo tenemos entonces si sustituimos el 4 por ese valor ya lo tendrias!

Te pongo como lo haría yo y comparas y si tienes alguna duda lo vemos!
```suggestion
let result = "";

for (let i = 0; i < hacker1.length; i++){
  result += `${hacker1[i]} `;
}
result = result.trim();
console.log(result.toUpperCase());
```

El trim que he puesto te explico, es un metodo que lo que hace es quitarte los espacios por delante y por detras! asi te curas de problemas :D, te pongo la docu por aqui
https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/String/trim